lcls_tools/common/measurements/emittance_measurement.py

```diff
@@ -0,0 +1,79 @@
+from lcls_tools.common.devices.reader import create_magnet, create_screen, create_wire
+from lcls_tools.common.devices.magnet import MagnetCollection
+from lcls_tools.common.devices.device import Device
+from lcls_tools.common.measurements.measurement import Measurement
+
+from meme.model import Model
+
+class QuadScanEmittance(Measurement):
+    model_beamline: str
```

Comment:
Instead of passing model_beamline, magnet_area, magnet_name, I would define a field 
`magnet_collection: MagnetCollection`, same for the measurement_device class (which will have the area attribute inside of it). These can be public attributes that get dumped with everything else since they are pydantic objects

Reply:
these fields should be required

Reply:
We can also make it so that we can pass a string to these attributes, for that you would use the following code for example in the magnet_collection class
```python
@field_validator("magnet_collection", mode="before")
def validate_magnet_collection(cls, value):
    if isinstance(value, str):
        value = create_magnet_collection(value)
    
    return value
```